plasmapy/formulary/parameters.py

```diff
@@ -801,15 +801,18 @@ def Hall_parameter(
     Parameters
```

Comment:
I felt there was still a lot to be done to clarify what is being calculated and how the function's arguments link to the Hall parameter expression, so I tried to make this more explicit.  I also updated links between referenced functionality.  So, here's my take at what the docstring should look like...

```python
r"""
Calculate the ``particle`` Hall parameter for a plasma.

The Hall parameter for plasma species :math:`s` (``particle``) is given by:

.. math::

    \beta_{s} = \frac{\Omega_{c s}}{\nu_{s s^{\prime}}}

where :math:`\Omega_{c s}` is the gyrofrequncy for plasma species :math:`s`
(``particle``) and :math:`\nu_{s s^{\prime}}` is the collison frequency
between plasma species :math:`s` (``particle``) and species
:math:`s^{\prime}` (``ion``).

**Aliases:** `betaH_`

Parameters
----------
n : `~astropy.units.quantity.Quantity`
    The number density associated with ``particle``.
T : `~astropy.units.quantity.Quantity`
    The temperature of associated with ``particle``.
B : `~astropy.units.quantity.Quantity`
    The magnetic field.
ion : `~plasmapy.particles.Particle`
    The type of ion ``particle`` is colliding.
particle : `~plasmapy.particles.Particle`
    The particle species for which the Hall parameter is calculated for.
    Representation of the particle species (e.g., 'p' for protons, 'D+' for
    deuterium, or 'He-4 +1' for singly ionized helium-4). If no charge state
    information is provided, then the particles are assumed to be singly
    charged.
coulomb_log : `float`, optional
    Preset value for the Coulomb logarithm. Used mostly for testing purposes.
V : `~astropy.units.quantity.Quantity`
    The relative velocity between ``particle`` and ``ion``.  If not provided,
    then the ``particle`` thermal velocity is assumed
    (`~plasmapy.formulary.parameters.thermal_speed`).
coulomb_log_method : `str`, optional
    Method used for Coulomb logarithm calculation. (see
    `~plasmapy.formulary.collisions.Coulomb_logarithm`)

See Also
--------
plasmapy.formulary.parameters.gyrofrequency
plasmapy.formulary.collisions.fundamental_electron_collision_freq
plasmapy.formulary.collisions.fundamental_ion_collision_freq
plasmapy.formulary.collisions.Coulomb_logarithm

Returns
-------
`~astropy.units.quantity.Quantity`
    Hall parameter for ``particle``.

Notes
-----
* For calculating the collision frequency,
  `~plamsapy.formulary.colisions.fundamental_electron_collision_freq` is used
  when ``particle`` is an electron and
  `~plamsapy.formulary.colisions.fundamental_ion_collision_freq` when
  ``particle`` is an ion.
* The collision frequencies are calculated assuming a slowly moving
  Maxwellian distribution.

Examples
--------
>>> from astropy import units as u
>>> Hall_parameter(1e10 * u.m**-3, 2.8e3 * u.eV, 2.3 * u.T, 'He-4 +1', 'e-')
<Quantity 7.26446...e+16>
>>> Hall_parameter(1e10 * u.m**-3, 5.8e3 * u.eV, 2.3 * u.T, 'He-4 +1', 'e-')
<Quantity 2.11158...e+17>
"""
```

Reply:
My suggested docstring will render like this in the documentation...

<img width="559" alt="image" src="https://user-images.githubusercontent.com/29869348/102108373-af235480-3de7-11eb-879b-5d0a54b26f53.png">